rotkehlchen/chain/evm/frontend_structures/protocols.py

```diff
@@ -0,0 +1,224 @@
+from rotkehlchen.chain.ethereum.decoding.constants import CPT_GNOSIS_CHAIN
```

Comment:
As I said elsewhere this should move to the decoders themselves. So this is againt not a frontend specific thing and should cease to exist as a file.